inc/class-markdownparser.php

```diff
@@ -23,6 +23,51 @@ public function __construct( Page $page ) {
 		$this->current_page = $page;
 	}
 
+	/**
+	 * Parse an inline image.
+	 *
+	 * Overridden to resolve relative URLs based on the current page.
+	 *
+	 * @param mixed $data
+	 * @return array
+	 */
+	protected function inlineImage( $data ) {
+		$result = parent::inlineImage( $data );
+		if ( empty( $result ) ) {
+			return $result;
+		}
+
+		// Re-parse the link data.
+		$link = $data;
+		$link['text'] = substr( $link['text'], 1 );
+		$link_result = parent::inlineLink( $link );
+		if ( empty( $link_result ) ) {
+			return $result;
+		}
+
+		$src = $link_result['element']['attributes']['href'];
```

Comment:
is there ever a chance a link could be created without an `href` attribute? Like an anchor, or direct addition of HTML?

Reply:
tl;dr no: https://github.com/erusev/parsedown/blob/master/Parsedown.php#L1399